snip-gui/src/app/Services/api-requests.service.ts

```diff
@@ -171,7 +184,9 @@ export class ApiRequestsService {
   private handleError<T>(Operation = 'operation', result?: T)
```

Comment:
If you want to handle errors and suppress them here, use the snackbar to show them. Or, just remove the "CatchError" and let the components use the snackbar